music/urls.py

```diff
@@ -24,6 +25,7 @@
     path('/station', StationView.as_view()),
     path('/station/theme', ThemeView.as_view()),
     path('/station/theme<int:theme_id>', StationThemeView.as_view()),
+    path('/theme<int:theme_id>', UserThemeView.as_view()),
```

Comment:
27라인도 슬러시 붙여서 id 주는 방식으로 바꿔주세요. 그렇게 하지않으면 프론트에서 id문자열로 붙여줘야하고 RESTful API 형식에도 맞지않습니다.
```suggestion
    path('/theme/<int:theme_id>', UserThemeView.as_view()),
```